vllm/transformers_utils/config.py

```diff
@@ -65,16 +75,24 @@ def get_config(
         kwargs["gguf_file"] = Path(model).name
         model = Path(model).parent
 
-    try:
-        if load_from_params:
-            config = load_params_config(model, revision)
+    if config_format == ConfigFormat.AUTO:
+        if file_exists(model, "config.json", revision=revision, token=kwargs.get("token")):
+            config_format = ConfigFormat.HF
         else:
+            config_format = ConfigFormat.MISTRAL
```

Comment:
Could you make this an elif where you check for the mistral config, and the else is a failure/warning where we default to HF? I don't think we want to assume something is a mistral config simply if there is no config.json - I'm not sure if HF always requires a specifically named config.json

Reply:
Yeah makes sense - HF does for now always require a config.json (see here: https://github.com/huggingface/transformers/blob/21fac7abba2a37fae86106f87fcf9974fd1e3830/src/transformers/utils/__init__.py#L245)

(can also import this variable)

But yeah will also check for mistral config and else throw an error

Reply:
Made the change - let me know what you think :-)

Reply:
Perfect!